core/src/main/java/hudson/Util.java

```diff
@@ -1805,16 +1805,4 @@ public static long daysElapsedSince(@NonNull Date date){
     private static PathRemover newPathRemover(@NonNull PathRemover.PathChecker pathChecker) {
         return PathRemover.newFilteredRobustRemover(pathChecker, DELETION_RETRIES, GC_AFTER_FAILED_DELETE, WAIT_BETWEEN_DELETION_RETRIES);
     }
-
-    /**
-     * If this flag is true, native implementations of {@link FilePath#chmod}
-     * and {@link hudson.util.IOUtils#mode} are used instead of NIO.
-     * <p>
-     * This should only be enabled if the setgid/setuid/sticky bits are
-     * intentionally set on the Jenkins installation and they are being
-     * overwritten by Jenkins erroneously.
```

Comment:
Is this still relevant? (I think we can probably go without, but looks like it deserves some investigation if you haven't yet.)

Reply:
Quoth Devin in [JENKINS-46725 (comment)](https://issues.jenkins.io/browse/JENKINS-46725?focusedCommentId=341691&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-341691):

> Yes, `hudson.Util#NATIVE_CHMOD_MODE` should be able to be removed. It was added to Jenkins 2.93 as part of [JENKINS-36088](https://issues.jenkins.io/browse/JENKINS-36088) to help remove unnecessary usages of native code. There was a small compatibility concern with the Java vs native implementations so I defaulted to the Java impl and added a system property to enable the native implementation, but I have not seen any reports from users that needed to enable the native implementation so I do not anticipate any issues with removing the option.